test/models/album_test.rb

```diff
@@ -0,0 +1,31 @@
+require 'test_helper'
+
+class AlbumTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+
+  # validates :artist, presence: true
+  # validates :name, presence: true
+
+  test 'album with no name is invalid' do
+    assert albums(:one).valid?
+    albums(:one).name = nil
+    assert_not albums(:one).valid?
+  end
+
+  test 'album with no artist is invalid' do
+    assert albums(:one).valid?
+    albums(:one).artist = nil
+    assert_not albums(:one).valid?
+  end
+
+  test 'an album must have a unique name' do
+   album = Album.new(album: {name: one})
```

Comment:
We will need to provide an `artist` as well, and not wrap this in an `album` hash (which we only need to do for controller params):
```ruby
album = Album.new(name: 'A Name', artist: 'An Artist')
```